### DISCUSSION TITLE
Rhapsody BCR+TCR full length data input

### DISCUSSION THREAD

**Discussion by albertopeloso** [milaboratory/mixcr#1702] [General]
Hi,
we are trying to use MiXCR with our single-cell WTA+TCR+BCR full length data. 
We have 4 libraries (SampleTag+ABseq, WTA, TCR, BCR), and we pooled together 4 samples in 2 Illumina S4 lanes. So we ended up with 1 set of fastqs per lane (pool1 and pool2), and SampleTag_ABseq.fastq, WTA.fastq, BCR.fastq, TCR.fastq.
Which would be the best option to load fastqs in MiXCR? Just to be sure to go with the right preset. We saw that in the presets guide fastqs are divided per sample, so do we need to re-build them to use with the pipeline?
Thanks a lot!

**Comment by mizraelson** [comment]
Hi, currently we would need to have a table with Sample Name and CELL barcodes associated with it to process such data. The full support of Sample Tags when you will be able to just use the SampleTag_ABseq.fastq file is coming soon. If you can share the data with us I can help you with generating the Sample CELL table and run the analysis. Please write to support@milaboratories.com

**Comment by albertopeloso** [comment]
Hi and thanks a lot for the reply! I wrote an email to your support address, attaching our Sample Name and Cell Barcode table.
